### PR TITLE
Edited README to reflect V3

### PR DESCRIPTION
**Pull request by CU-0xff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,23 +34,27 @@ DeepCode uses symbolic AI to process hundreds of millions of commits in open sou
...
-## Video on how to install the extension
+  <a href="https://youtu.be/3J5cVuEJ8WE&utm_source=vscode-extension-readme" target="_blank">![](images/install-and-use-vs-code-extension.png)</a>
 
-- We've prepared a 2 minutes video on how to install the extension. Head over to YouTube to quickly get the extension up and running:
+# Installation
 
-  <a href="https://www.youtube.com/watch?v=Cfe4OMvlfpc&utm_source=vscode-extension-readme" target="_blank">![](images/how-to-install-vs-code-extension.png)</a>
+In a nutshell, you can either use navigate to [DeepCode Extension on the Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=DeepCode.deepcode) and press _Install_ or use the build in mechanism in Visual Studio Code behind the _Extensions Icon_ ![Extensions Icon](images/extension_icon.png) in the sidebar. 
 
 ## Tips on the installation process
```
Comment (deepcodeg) [inline README.md:53]: @CU-0xff pr looks great, but please remove this section, users no longer have to choose between both. Thanks for the pr :)

File: README.md [hunk 2]
```diff
@@ -34,23 +34,27 @@ DeepCode uses symbolic AI to process hundreds of millions of commits in open sou
...
 
 ## Supported languages
 
-JavaScript, TypeScript, Java, C/C++, and Python are currently supported.
+JavaScript, TypeScript, Java, C/C++ (beta), and Python are currently supported. We also provide specific coverage for VUE and REACT.
 
-# Installation
+## Video on how to install and use the extension
+
+- We've prepared a 2 minutes video on how to install and use the extension. Head over to YouTube to quickly get the extension up and running:
```
Comment (deepcodeg) [inline README.md:45]: @CU-0xff btw video is a bit longer than 2 minutes ;)

File: README.md (hunk 1 above)
Comment (CU-0xff) [reply README.md:53]: I explained the basic idea better. I like to have the direct hyperlink to the marketplace as it reduces the install to two clicks, while the build in marketplace needs so searching.

File: README.md (hunk 2 above)
Comment (CU-0xff) [reply README.md]: True :-) I changed to short :-) In the video comments I provided a clickable ToC - so it should be easy to navigate.

File: README.md (hunk 1 above)
Comment (deepcodeg) [reply README.md:53]: My comment was on "## Tips on the installation process" section as it is not valid anymore. Does your version have a choice between cloud and self-managed?

**Review by deepcodeg — APPROVED** [review]

**Pull request merged**